diff --git a/src/parser/gjs-gts-parser.js b/src/parser/gjs-gts-parser.js
--- a/src/parser/gjs-gts-parser.js
+++ b/src/parser/gjs-gts-parser.js
@@ -21,7 +21,11 @@
 try {
   babelParser = require('@babel/eslint-parser/experimental-worker');
 } catch {
-  // optional peer; left null
+  try {
+    babelParser = require('@babel/eslint-parser');
+  } catch {
+    // optional peer; left null
+  }
 }

 /**
